video-templates/scripts/cf-source-youtube.js

```diff
@@ -0,0 +1,63 @@
+var $ = jQuery,
```

Comment:
@royboy789 Why this versus noconflict wrapper? Just curious, assuming I'm right that var declaration keeps in scope of this file, and therefore you're not messing with $ in other scopes, then whatever, just curious.